docs/getting_started/setting_up_an_api.md

```diff
@@ -0,0 +1,62 @@
+Instead, if you are storing chemical formulae as an unreduced count per simulation cell, e.g., `"Si4O8"`, then it is impossible to remap the filter `chemical_formula_reduced="O2Si"` such that it matches all structures with the correct formula unit (e.g., `"SiO2"`, `"Si2O4"`, ...).
+This would then instead require option 2 above, namely either the addition of auxiliary fields that store the correct (or mappable) OPTIMADE format in the database, or the creation of a secondary database that returns the pre-converted structures.
+
+In the simplest case, the mapper classes can be used to define aliases between fields in the database and the OPTIMADE field name; these can be configured via the [`aliases`][optimade.server.config.ServerConfig.aliases] option as a dictionary mapping stored in a dictionary under the appropriate endpoint name, e.g. `"aliases": {"structures": {"chemical_formula_reduced": "my_chem_form"}}`, or defined as part of a custom mapper class.
+
+In either option, you should now be able to insert your data into the corresponding MongoDB (or otherwise) collection.
+
+## Serving custom fields/properties
+
+According to the OPTIMADE specification, any field not standardized in the specification must be prefixed with an appropriate "provider prefix" (e.g., "`_aflow`" for [AFLOW](https://aflow.org) and "`_cod`" for [COD](https://crystallography.net)).
+This prefix is intended to be unique across all [OPTIMADE providers](https://github.com/Materials-Consortia/providers) to enable filters to work across different implementations.
+The prefix can be set in the [configuration](../configuration.md) as part of the [`provider`][optimade.server.config.ServerConfig.provider] option.
+
+Once the prefix has been set, custom fields can be listed by endpoint in the [`provider_fields`][optimade.server.config.ServerConfig.provider_fields] configuration option.
+Filters that use the prefixed form of these fields will then be passed through to the underlying database without the prefix, and then the prefix will be reinstated in the response.
+
+It is recommended that you provide a description, type and unit for each custom field that can be returned at the corresponding `/info/<entry_type>` endpoint.
+To do this, the underlying `EntryResourceAttributes` model will need to be sub-classed, the pydantic fields added to that class, and the server adjusted to make use of those models in responses.
+The reference server is currently not flexible enough to use custom response classes, so instead the code will need to be forked and modified for your implementation.
```

Comment:
Following from #925, I should probably bulk out this section a bit.

Reply:
The hole pydantic alias thingy, should definitely be mentioned here.

Reply:
Added the snippet from our discussion. Will hopefully make this approach much easier in a future PR (as discussed in #929).